actions/recaptcha_solver.js

```diff
@@ -26,6 +26,9 @@ exports.recaptchaSolver = async function recaptchaSolver(page, request) {
         recaptchaData = await page.findRecaptchas();
     }
 
+    if (request.body.navigationOptions) {
+        await page.waitForNavigation(request.body.navigationOptions);
+    }
```

Comment:
Not sure if we should await for both solveRecaptchas and waitForNavigation, as with indirect navigation on clicks